packages/curriculum-compiler-json/index.sandbox.js

```diff
@@ -0,0 +1,116 @@
+// Use this sandbox to play with the parser
+const parser = require('../../packages/curriculum-parser/index')
```

Comment:
You should add an actual example here that uses your compiler, not the parser

Reply:
gotcha